### PR TITLE
Create Homework_1_SergioFG

### PR DESCRIPTION
**Pull request by Serg1oFG**
Re-uploaded my HW file

### CODE REVIEW

File: Homework_1_SergioFG [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+data_iris.head
+fig = px.scatter(data_iris, x="sepal_width", y="Sepal_Length", color="class", error_x="e", error_y="e")
+fig.show()
+
+
+##################################################################################################
+#                                          bar graph                                             #
+##################################################################################################
+
+# read in original data --> without the "e"
```
Comment (nikitz77) [inline Homework_1_SergioFG:46]: Ha! This makes sense. Yep

File: Homework_1_SergioFG [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+X_train, X_test, y_train, y_test = train_test_split(X_trans, y, random_state=0)
+pipe = Pipeline([('scaler', Normalizer()), ('svc', SVC())])
+
+# The pipeline can be used as any other estimator
+# and avoids leaking the test set into the train set
+### I split the data to avoid data leackage. Not that it matters but it runs :p 
+
+pipe.fit(X_train, y_train)
+ 
+pipe.score(X_test, y_test)
```
Comment (nikitz77) [inline Homework_1_SergioFG:114]: Might be helpful to save your end results.  Otherwise looks good

**Review by nikitz77 — APPROVED** [review]
Please review my comments below. I find no issues so far. Code was pretty similar to what I have. Just please consider looking into having one or two results to be able to compare numbers/see progress

File: Homework_1_SergioFG (hunk 2 above)
Comment (Serg1oFG) [reply Homework_1_SergioFG:114]: Will do so.

**Pull request merged**